src/styles.scss

```diff
@@ -29,4 +29,12 @@ body {
 @keyframes spin {
     0% { transform: rotate(0deg); }
     100% { transform: rotate(360deg); }
+}
+
+form{
```

Comment:
This shouldn't be global styles add to the styles of the component itself, which should be `timer-admin.component.scss`